test/plugins/navigation/next/linearNextItemWarning/test.js

```diff
@@ -45,68 +45,52 @@ define([
         );
     });
 
-    pluginApi = [
-        {
-            name: 'init',
-            title: 'init'
-        },
-        {
-            name: 'render',
-            title: 'render'
-        },
-        {
-            name: 'finish',
-            title: 'finish'
-        },
-        {
-            name: 'destroy',
-            title: 'destroy'
-        },
-        {
-            name: 'trigger',
-            title: 'trigger'
-        },
-        {
-            name: 'getTestRunner',
-            title: 'getTestRunner'
-        },
-        {
-            name: 'getAreaBroker',
-            title: 'getAreaBroker'
-        },
-        {
-            name: 'getConfig',
-            title: 'getConfig'
-        },
-        {
-            name: 'setConfig',
-            title: 'setConfig'
-        },
-        {
-            name: 'getState',
-            title: 'getState'
-        },
-        {
-            name: 'setState',
-            title: 'setState'
-        },
-        {
-            name: 'show',
-            title: 'show'
-        },
-        {
-            name: 'hide',
-            title: 'hide'
-        },
-        {
-            name: 'enable',
-            title: 'enable'
-        },
-        {
-            name: 'disable',
-            title: 'disable'
-        }
-    ];
+    pluginApi = [{
```

Comment:
Nothing added, just reformat. We should have a common format based on prettier...

Reply:
I'm agree, but in the meantime the selected style remains 1TBS https://oat-sa.atlassian.net/wiki/spaces/DEV/pages/79037064/TAO+Frontend+Coding+guide#TAOFrontendCodingguide-Generalrules